src/app/clusters/service-area-server/service-area-delegate.h

```diff
@@ -61,12 +51,12 @@ class Delegate
      */
     virtual CHIP_ERROR Init() { return CHIP_NO_ERROR; };
 
-    //*************************************************************************
-    // Command handling support
-
     /**
      * @brief Can the selected locations be set by the client in the current operating mode?
-     * @param[out] statusText text describing why selected locations cannot be set (if return is false).
+     * @param[out] statusText text describing why the selected locations cannot be set (if return is false).
+     * Max size kMaxSizeStatusText.
+     * Note: If the return is false and statusText is not successfully edited, example due to a string that can be longer than
+     * kMaxSizeStatusText, the size of this value should be set to 0 with .reduce_size(0) to avoid memory leaks.
```

Comment:
```suggestion
     * Note: If the return is false and statusText is not successfully set, for example due to a string that can be longer than
     * kMaxSizeStatusText, the size of this value should be set to 0 with .reduce_size(0) to avoid callers using un-initialized memory.
```